### PR TITLE
Option to ignore parse file storage

### PR DESCRIPTION
**Pull request by OzgeAkin**
We already finished migrating files from parse S3 to our own S3 and we want to use only our file storage.
In the current version  of parse server, in the filesController, it points to two of the file storage based on the file name's prefix

We would like to have an option to config the server to point only to our S3 no matter what the prefix of the file names are.

We think this is important because eventually all the parsefile will be gone.

### CODE REVIEW

**Comment by natanrolnik** [comment]
@OzgeAkin How did you migrate your files from Parse's S3 to your own S3 bucket?

**Comment by OzgeAkin** [comment]
@natanrolnik By running a script which iterates through json file to find each parse file object and uploading it to our s3 bucket.

File: src/ParseServer.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ class ParseServer {
     maxUploadSize = '20mb',
     verifyUserEmails = false,
     preventLoginWithUnverifiedEmail = false,
+      useOnlyFilesAdapter = false,
```
Comment (flovilmart) [inline src/ParseServer.js]: nit: alignment

**Comment by facebook-github-bot-3** [comment]
@OzgeAkin updated the pull request.

**Comment by facebook-github-bot-7** [comment]
@OzgeAkin updated the pull request.

**Comment by facebook-github-bot-3** [comment]
@OzgeAkin updated the pull request.

**Comment by facebook-github-bot-3** [comment]
@OzgeAkin updated the pull request.

**Comment by drew-gross** [comment]
Rather than adding a new option, what about checking for the existence of a file key? If one is not provided, we can skip the check for files on Parse.com

**Comment by OzgeAkin** [comment]
@drew-gross That is a good suggestion. Right now the default value for fileKey is 'invalid-file-key', I could have done an if statement in FilesController that checks if the file key equals to this value. However, can you assure that this value would not change in the future?  Or should i check for multiple values such as 'invalid-file-key', '', '-1' and etc.

**Comment by drew-gross** [comment]
I'd suggest removing the default. Then if `fileKey` is `undefined` you can skip the check.

**Comment by facebook-github-bot-2** [comment]
@OzgeAkin updated the pull request.

**Comment by facebook-github-bot-0** [comment]
@OzgeAkin updated the pull request.

**Comment by facebook-github-bot-8** [comment]
@OzgeAkin updated the pull request.

**Comment by OzgeAkin** [comment]
@drew-gross I made the changes according to your suggestion. Thanks.

**Pull request merged**